package.json

```diff
@@ -14,7 +14,8 @@
     "prepublish": "npm test",
     "postinstall": "npm run babel",
     "babel": "babel lib -d dest",
-    "coveralls": "nyc report --reporter=text-lcov | coveralls"
+    "coveralls": "nyc report --reporter=text-lcov | coveralls",
+    "precommit": "npm run lint && npm run test"
```

Comment:
the `precommit` script should be just `npm test`. `npm run lint` runs anyway as `pretest`